egs2/snips/asr1/conf/tuning/train_asr_transformer_adam.yaml

```diff
@@ -0,0 +1,31 @@
+# network architecture
```

Comment:
At least for the initial PR, we could stick to one working configuration for the recipe, so please remove those not working